### PR TITLE
PR Owners can change their minds

### PR DESCRIPTION
**Pull request by JordanReiter**
If a PR owner changes their mind & votes against the pull request, that vote should count.
Also I went ahead and switched to a gender-neutral pronoun.

### CODE REVIEW

**Review by pchauncey — CHANGES_REQUESTED** [review]
Please also update the README.md to reflect this change?

File: github_api/voting.py [hunk 1]
```diff
@@ -31,8 +31,10 @@ def get_votes(api, urn, pr):
         if vote and vote_owner != pr_owner:
             votes[vote_owner] = vote
 
-    # by virtue of creating the PR, the owner casts his vote as 1
-    votes[pr_owner] = 1
+    # by virtue of creating the PR, the owner casts their vote as 1
+    # unless they later vote against it, in which case we respect that
+    if votes.get('pr_owner') != -1:
```
Comment (YBadiss) [inline github_api/voting.py]: this is incorrect, should be `votes.get(pr_owner)`

Comment (Ajedi32) [reply github_api/voting.py]: Thanks, I was just about to point that out. 

This line in its current form actually just allows the GitHub user named "pr_owner" to override the actual PR owner's vote.

Also, I feel it's worth pointing out that as of the moment of this review, there are 23 other people who voted for this PR without noticing this mistake.

**Comment by Ajedi32** [comment]
Can't the PR owner already just veto the PR if they choose by closing it themselves?

**Comment by JordanReiter** [comment]
@Ajedi32 probably but they may as well still have the option. Maybe they end up with misgivings about the pull request but still want it to be decided by a vote.

**Review by pchauncey — APPROVED** [review]

File: github_api/voting.py (hunk 1 above)
Comment (JordanReiter) [reply github_api/voting.py]: OMG This thing needs to have tests… that's going to be interesting.

**Comment by patricknelson** [comment]
Related: #82 PR owners can submit code that was not what was voted for and use it as a method to inject unexpected malicious code.

**Comment by rudehn** [comment]
Related #48 somehow got through.

**Comment by viktorsec** [comment]
@JordanReiter please resolve conflicts

**Comment by patricknelson** [comment]
Oh... oh my god... it's conscious and has an attitude. WE'RE DOOMED :robot:

**Pull request closed without merging**